### PR TITLE
ci: store kopia logs as GitHub artifacts

### PR DESCRIPTION
**Pull request by jkowalski**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -23,3 +23,4 @@ node_modules/
 .boto
 *.log
 *~
+.logs/
```
Comment (julio-lopez) [inline .gitignore:26]:
```suggestion
.logs/

```

File: .github/workflows/make.yml [hunk 2]
```diff
@@ -105,3 +105,10 @@ jobs:
       run: make travis-setup
     - name: Build
       run: make travis-release
+    - name: Store Kopia Logs
+      uses: actions/upload-artifact@v2
+      if: ${{ always() }}
+      with:
+        name: kopia-logs
```
Comment (julio-lopez) [inline .github/workflows/make.yml:112]: Would logs be uploaded after every build / PR / merge / release?
Any chance anything could be accidentally leaked?

Comment (jkowalski) [reply .github/workflows/make.yml:112]: those are logs from integration tests only, we don't have sensitive stuff there, do we?

Comment (jkowalski) [reply .github/workflows/make.yml:112]: Also access appears to be limited to those with write access, which seems sane:

Per: https://github.com/actions/upload-artifact#where-does-the-upload-go

> This icon will only appear for users who have write permissions to the repository.

Comment (jkowalski) [reply .github/workflows/make.yml:112]: My bad, this appears to be permission to delete the artifacts.

Also non-logged in users don't see the artifacts at all, but I don't have any other GH user that's not in the repo to try and see if they have read access.

Comment (julio-lopez) [reply .github/workflows/make.yml:112]: FWIW, I don't see the icon for the artifacts.

**Comment by jkowalski** [comment]
closing this for now

**Pull request closed without merging**